### PR TITLE
feat(workflow): Sort group tags alphabetically

### PR DESCRIPTION
**Pull request by scttcper**
When the page is loaded the backend sends them in `-count` order which can seem almost random. This sorts them by tag name.

- **sort tags alphabetically**
- switch to RTL from enzyme
- use AsyncComponent instead of custom component
- removes some inline styles

image

### CODE REVIEW

File: static/app/views/organizationGroupDetails/groupTags.tsx [hunk 1]
```diff
@@ -2,158 +2,124 @@ import * as React from 'react';
...
       <div>
-        <Container>{children}</Container>
+        {this.renderTags()}
         <Alert type="info">
           {tct(
             'Tags are automatically indexed for searching and breakdown charts. Learn how to [link: add custom tags to issues]',
             {
-              link: <a href={this.getTagsDocsUrl()} />,
+              link: (
+                <a href="https://docs.sentry.io/platform-redirect/?next=/enriching-events/tags" />
```
Comment (matejminar) [inline static/app/views/organizationGroupDetails/groupTags.tsx:121]: Could be `<ExternalLink/>`

File: static/app/views/organizationGroupDetails/groupTags.tsx [hunk 2]
```diff
@@ -2,158 +2,124 @@ import * as React from 'react';
...
-                </DetailsLinkWrapper>
-              </PanelHeader>
+              <StyledPanelHeader hasButtons>
+                <TagHeading>{tag.key}</TagHeading>
+                <GlobalSelectionLink
+                  className="btn btn-default btn-sm"
+                  to={`${baseUrl}tags/${tag.key}/`}
+                >
+                  {t('More Details')}
+                </GlobalSelectionLink>
```
Comment (matejminar) [inline static/app/views/organizationGroupDetails/groupTags.tsx:73]: I think I used something like this previously:
```
<Button
    to={{
      pathname: `/organizations/${orgSlug}/something`,
      query: extractSelectionParameters(location.query),
    }}
 >
```

**Review by matejminar — APPROVED** [review]
Looks good, left two minor suggestions

File: static/app/views/organizationGroupDetails/groupTags.tsx (hunk 2 above)
Comment (scttcper) [reply static/app/views/organizationGroupDetails/groupTags.tsx:73]: thanks!!

**Pull request merged**